bears/vcs/git/GitCommitBear.py

```diff
@@ -105,8 +105,12 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
         stdout = stdout.rstrip('\n')
         pos = stdout.find('\n')
         shortlog = stdout[:pos] if pos != -1 else stdout
+        if pos != -1:
+            if stdout[pos+2] == '\n':
+                yield Result(self, 'Extra Newlines found.')
```

Comment:
but if i return it; then won't the code be errorneous?
because i will be returning before defining what 'body' is.

and 'body' is used later on in the code :worried: